junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/LauncherFactory.java

```diff
@@ -54,10 +59,29 @@ private LauncherFactory() {
 	 * @throws PreconditionViolationException if no test engines are detected
 	 */
 	public static Launcher create() throws PreconditionViolationException {
-		Launcher launcher = new DefaultLauncher(new ServiceLoaderTestEngineRegistry().loadTestEngines());
-		for (TestExecutionListener listener : new ServiceLoaderTestExecutionListenerRegistry().loadListeners()) {
-			launcher.registerTestExecutionListeners(listener);
+		return create(LauncherConfig.builder().build());
+	}
+
+	/**
+	 * Factory method for creating a new {@link Launcher} using dynamically
+	 * detected test engines.
```

Comment:
That's no longer true. Test engines will only be auto-registered if the supplied config specifies that.

Please reword.